src/microbe_stage/editor/TolerancesEditorSubComponent.cs

```diff
@@ -672,8 +692,28 @@ private void TryApplyPressureChange(float min, float max)
 
         if (!TriggerChangeIfPossible())
         {
-            pressureMinSlider.Value = CurrentTolerances.PressureMinimum;
-            pressureMaxSlider.Value = CurrentTolerances.PressureMaximum;
+            if (keepSamePressureFlexibility)
+            {
+                var pressureRange = max - min;
+
+                var extremeMin = CalculateSliderExtremeValue(Constants.TOLERANCE_CHANGE_MP_PER_PRESSURE,
+                    min, CurrentTolerances.PressureMinimum);
+
+                var extremeMax = extremeMin + pressureRange;
+
+                reusableTolerances.PressureMinimum = extremeMin;
+                reusableTolerances.PressureMaximum = extremeMax;
+
+                pressureMinSlider.Value = extremeMin;
+                pressureMaxSlider.Value = extremeMax;
+            }
```

Comment:
Doesn't this need an else branch? Otherwise this PR is almost complete but not exactly.